tests/org.eclipse.xtend.core.tests/src/org/eclipse/xtend/core/tests/typesystem/BatchLinkingServiceTest.xtend

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************************
+ * Copyright (c) 2015 itemis AG (http://www.itemis.eu) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *******************************************************************************/
+package org.eclipse.xtend.core.tests.typesystem
+
+import org.eclipse.xtend.core.tests.AbstractXtendTestCase
+import org.junit.Test
+
+/**
+ * @author efftinge - Initial contribution and API
+ */
+class BatchLinkingServiceTest extends AbstractXtendTestCase {
+	
+	@Test def void testBug464762() {
```

Comment:
Could you please move the method to org.eclipse.xtend.core.tests.linking.LinkingErrorTest or add this new test class to the LinkingSuite?